PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/AppMonet+MoPub.m

```diff
@@ -22,15 +22,113 @@
 
 #import "AppMonet+MoPub.h"
 #import <HyBid/HyBid.h>
+#import "MPAdView.h"
+
+@interface AppMonet ()
+typedef struct { MPAdView *adView; void (^onReadyBlock)(void); } AppMonetAdStruct;
+@property (class, nonatomic) NSMutableDictionary *dict;
+@end
 
 @implementation AppMonet (MoPub)
 
-+ (void)addNativeBids:(id)adRequest andAdUnitId:(NSString *)adUnitId andTimeout:(NSNumber *)timeout :(void (^)(void))onReadyBlock {
++ (void)addBids:(MPAdView *)adView andAppMonetAdUnitId:(NSString *)appMonetAdUinitId andTimeout:(NSNumber *)timeout :(void (^)(void))onReadyBlock {
```

Comment:
there should be one method to put bids on interstitial as well, no?

Reply:
ahh correct 👍